modules/ding_place2book/ding_place2book.fields.inc

```diff
@@ -551,11 +558,19 @@ function _ding_place2book_sync_p2b_entities($entity, array $settings) {
       else {
         $event = $p2b->updateEvent($event_maker_id, $event_id, $options);
       }
+
+      $items = array(
+        array(
+          'event_id' => $event->id,
+          'event_maker_id' => $event_maker_id,
+          'synchronize' => 1,
+        ),
+      );
     }
     catch (Exception $ex) {
       watchdog_exception('ding_place2book', $ex);
       drupal_set_message($ex->getMessage(), 'error');
-      return $items;
+      return FALSE;
```

Comment:
This seems to indicate that the DocBlock should be updated or that we should just make it explicit that returning an empty array means that no items are sync'ed.